Apps/APN/.gitignore

```diff
@@ -0,0 +1,30 @@
+app_store_connect_creds.json
+ami_app_ci_server-google_cloud_service_account.json
+gc_keys.json
+GoogleService-Info.plist
+node_modules
+.DS_Store
+env.js
+fastlane/report.xml
+
+# iOS
+iOS/build
+
+# Android/IntelliJ
+#
+android/app/build
+android/build/
+.idea
+.gradle
+local.properties
+*.iml
+
+# Google-services.json 
+android/app/google-services.json
+
+# Gradle files
+.gradle/
+build/
+
+# CocoaPods
+/ios/Pods/
```

Comment:
I suggest that we move this `.gitignore` file into `Apps/` directory so it applies to all of our sample apps, not just for this 1 sample app. 

Thanks for adding it!

Reply:
Moved to `Apps/` in [this commit](https://github.com/customerio/customerio-reactnative/pull/136/commits/7d0fc164ba11d16dcb4424e8da83bc80cf531055)